src/lobpcg.jl

```diff
@@ -869,9 +872,9 @@ function lobpcg!(iterator::LOBPCGIterator; log=false, tol=nothing, maxiter=200,
     end
     n = size(X, 1)
     sizeX = size(X, 2)
-    residualTolerance = (tol isa Void) ? (eps(real(T)))^(real(T)(4)/10) : real(tol)
+    residualTolerance = (tol isa Nothing) ? (eps(real(T)))^(real(T)(4)/10) : real(tol)
```

Comment:
Could the problem be here? @mohamed82008 
Tests may fail since
```
(eps(real(T)))^(real(T)(4)/10) > sqrt(eps(real(T)))
```
but the test tolerance is set to `sqrt(eps(real(T)))`.

Reply:
Nevermind, `tol` is actually not a `Nothing` in tests: it is passed in and matches the test tolerance.

Reply:
LOBPCG tests pass on my machine.

Reply:
Oh I see you changed the seed.

Reply:
It also runs fine on my machine (yes, with a modified seed), but apparently not on Travis.